app/views/comments/index.html.erb

```diff
@@ -9,7 +9,7 @@
                 <% @comments.each do |comment| %>
                   <tr>
                     <td>
-                      <%= link_to comment_path(comment.id), class: 'text-dark' do %> <%= comment.text %><% end %>
+                      <%= link_to comment_path(comment.id), class: 'text-dark link' do %> <%= comment.text %><% end %>
```

Comment:
Please use a definitive class name, like `link` is very generalized.
You can use `comment-text-with-limit`